### ISSUE TITLE
Passing JavaScript events through chart options

### ISSUE THREAD

**Issue by andreibondarev** [styd/apexcharts.rb#45]
Is it possible to pass in events through the options?https://apexcharts.com/docs/options/chart/events/

I have the following code, but it doesn't work:
```
<%=
    events: {
      touchstart: "function(event, chartContext, config) { console.log('Test') }"
    }
%>
```

**Comment by styd** [comment]
Once ApexCharts.JS supports passing function object like [this](https://github.com/apexcharts/apexcharts.js/issues/833#issuecomment-526822509), maybe you would be able to do so. In the meantime, maybe you can try using [functionable-json](https://github.com/styd/functionable-json) as I suggested [here](https://github.com/styd/apexcharts.rb/blob/master/README.md#formatter-function).

**Comment by styd** [comment]
From [this comment](https://github.com/apexcharts/apexcharts.js/issues/833#issuecomment-570922643), it seems that the feature for passing the args and body of a function in json is not going to be around soon because it needs to change code in many places. So I guess we just have to implement it in this gem.

**Comment by andreibondarev** [comment]
@styd I think what I ended up doing was just finding the chart instance in JS and settings the options there directly.

**Comment by styd** [comment]
Solved by #53 .
